_data/changelogs/component-date-range-picker.yml

```diff
@@ -2,6 +2,13 @@ title: Date range picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added Date range picker usability guidance.
+    summaryAdditional: Users should always be able to enter dates manually in Date range picker.
+    affectsGuidance: true
+    githubPr: 2822
+    githubRepo: uswds-site
+    versionUswds: N.N.N
```

Comment:
**issue:** Since this is a uswds-site update that isn't tied to a specific uswds release, we should remove the `versionUswds` data key:
```suggestion
```